### PR TITLE
Removed chart suffix from files

### LINKED ISSUE #199: Remove 'Chart' suffix from sc.chart modules
`sc.chart.primaryChart` -> `sc.chart.primary`
`sc.chart.rsiChart` -> `sc.chart.rsi`
etc.

### PR DESCRIPTION
**Pull request by djmiley**
Resolves #199

### CODE REVIEW

File: src/assets/js/app.js [hunk 1]
```diff
@@ -18,30 +18,30 @@
             viewDomain: []
         };
 
-        var primaryChart = sc.chart.primaryChart();
-        var secondaryChart = [];
+        var primary = sc.chart.primary();
```
Comment (jleft) [inline src/assets/js/app.js]: I'm not convinced about renaming these variables - I don't think it's as clear as it was previously. I wouldn't mind naming `navChart`, `navigator` though.

I'm also not sure about the event names - at this level in the application, is it as clear from the event name what's happening?

Comment (djmiley) [inline src/assets/js/app.js]: I agree, I'll have a look.

Comment (djmiley) [inline src/assets/js/app.js]: `navigator` is a reserved keyword

**Comment by djmiley** [comment]
> I'm not convinced about renaming these variables - I don't think it's as clear as it was previously. I wouldn't mind naming navChart, navigator though.

> I'm also not sure about the event names - at this level in the application, is it as clear from the event name what's happening?

`navigator` is a reserved keyword

File: src/assets/js/app.js (hunk 1 above)
Comment (jleft) [inline src/assets/js/app.js]: Cool, didn't know that!

File: src/assets/js/menu/primary/indicators.js [hunk 2]
```diff
@@ -20,7 +20,7 @@
                 dispatch.primaryChartIndicatorChange(indicator);
             });
 
-        var primaryChartSeriesMenu = function(selection) {
+        var primaryChartIndicatorMenu = function(selection) {
```
Comment (jleft) [inline src/assets/js/menu/primary/indicators.js]: Good spot

**Comment by jleft** [comment]
:+1:

**Pull request merged**